frontend/src/components/AppRatingCircle/index.vue

```diff
@@ -23,25 +23,16 @@ export default {
     return {
       cx: 40,
       cy: 40,
-      r: 25,
-      rating: 0
+      r: 25
     }
   },
   computed: {
     circumference() {
       return 2 * Math.PI * this.r
-    }
-  },
-  methods: {
-    calcRating() {
+    },
+    rating() {
```

Comment:
transition will only work with a starting value. I think the starting value needs to be this.circumfrence.

So the place it looks to rating it should instead default to rating || circumfrence.

Speaking of, given that value is undefined this will also run into errors without a check.